utils/math_functions.py

```diff
@@ -212,8 +212,8 @@ def direction(vector):
 
 
 def getPointBehindTheBall(point, theta):
-	x = point.x - (3.5 * BOT_RADIUS) * (math.cos(theta))
-	y = point.y - (3.5 * BOT_RADIUS) * (math.sin(theta))
+	x = point.x - (1*BOT_RADIUS) * (math.cos(theta))
+	y = point.y - (1*BOT_RADIUS) * (math.sin(theta))
```

Comment:
Is bot able to reach the point returned by this function without colliding with ball?

Reply:
He's using the getPointBehindBall, during intercept state, where defender is supposed to be ready to kick and clear the ball, I guess. Hence the small radius. But yeah we might have to increase it, if required.

Reply:
this function can also be used to find a point by the attacker to kick towards the goal. Now if bot is in between the region of ball and goal post, attacker needs to cross the ball hence a point at sufficient distance is required otherwise path planner replans the path at every instant assuming ball as an obstacle